perf/security/sds-tests/workload-sds/setup_test.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+NUM=${NUM:?"specify the number of httpbin and sleep workloads"}
+CLUSTER=${CLUSTER:?"specify the cluster for running the test"}
+
+if ! istioctl version; then 
+    exit
+fi
+
+timestamp=$(date +"%Y-%m-%d-%H-%M-%S")
+staticns=static-${timestamp}
```

Comment:
What is the differences between staticns and dynamicns (e.g., "static" versus "dynamic" applications deployed to these two namespaces)?

Reply:
In dynamicns, workloads are redeployed periodically (every 180s, which is hardcoded), in that way we delete and recreate proxy and SDS server, which is to catch any regression in the establishment of SDS flow.

One improvement in my mind is, if the workload pods are not healthy after a new deployment, we need to pause the redeployment, so that developers could inspect the pods at any time. We could add a health check logic in the config map.